builder/builder.go

```diff
@@ -174,12 +219,17 @@ func (b *Builder) convertAndBuild(ctx context.Context, target domain.Target, opt
 		dirIndex   = 0
 	)
 	var mts *states.MultiTarget
-	bf := func(childCtx context.Context, gwClient gwclient.Client) (*gwclient.Result, error) {
+	buildFn := func(childCtx context.Context, gwClient gwclient.Client) (*gwclient.Result, error) {
```

Comment:
I wanted to clarify this a bit as `bf` is a little cryptic for people new to the repo.

Reply:
seems reasonable; can we call it `buildFunc` to be consistent with the name that buildkit gives it:

```
func (c *Client) Build(ctx context.Context, opt SolveOpt, product string, buildFunc gateway.BuildFunc, statusChan chan *SolveStatus) (*SolveResponse, error) {
```